src/pcp/summary/pcp-summary.sh

```
@@ -163,21 +164,27 @@ else
     [ -z "$pcp_host" ] && pcp_host=`hostname`
 fi
 
-if eval pminfo $BATCH -f $metrics > $tmp/metrics 2>$tmp/err
+eval pminfo $BATCH -f $metrics > $tmp/metrics 2>$tmp/err
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)